{configure: true, fixed_frame_id: /camera_rgb_optical_frame, point_cloud_in: /camera/hd/points,
  use_image_filter: true, use_low_pass: false, voxelize: true, x_max: 4.0, x_min: -4.0,
  x_voxel: 0.005, xpix_max: 1612, xpix_min: 308, y_max: 4.0, y_min: -4.0, y_voxel: 0.005,
  ypix_max: 808, ypix_min: 272, z_max: 4.0, z_min: 0.5, z_voxel: 0.005}
